ui/src/app/base/components/NodeActionConfirmationText/NodeActionConfirmationText.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Node, NodeActions } from "app/store/types/node";
+import { getNodeActionTitle } from "app/store/utils";
+
+const NodeActionConfirmationText = ({
+  nodes,
+  action,
+}: {
+  nodes: Node[];
+  action: NodeActions;
+}): JSX.Element => (
+  <>
+    <p>
+      Are you sure you want to {getNodeActionTitle(action)}{" "}
```

Comment:
This isn't going to read well for most node actions, firstly because the util returns the action capitalised and secondly because of its position in the sentence.

e.g. this would currently return:
`Are you sure you want to Delete machine?` or
`Are you sure you want to Override failed testing machine?`

Either the prop should just be a `string` instead of `NodeActions`, or a different util should be used that's a bit more grammatically accurate. If you decide the latter you might be able to refactor parts of this util in `ActionForm`? https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/base/components/ActionForm/ActionForm.tsx#L10